test/stdlib/StringBridge.swift

```diff
@@ -56,6 +56,23 @@ StringBridgeTests.test("Tagged NSString") {
 #endif // not 32bit
 }
 
+StringBridgeTests.test("Constant NSString New SPI") {
+  if #available(6.1) {
+    //21 characters long so avoids _SmallString
+    let constantString:NSString = CFRunLoopMode.commonModes.rawValue as NSString
+    let regularBridged = constantString as String
+    let count = regularBridged.count
+    let bridged = String(
+      _immortalCocoaString: constantString,
+      count: count,
+      encoding: Unicode.ASCII.self
+    )
+    let reverseBridged = bridged as NSString
+    expectEqual(constantString, reverseBridged)
+    expectEqual(bridged, regularBridged)
```

Comment:
Are we meaning to test whether their contents are equal or whether their object addresses are equal?

Reply:
both are important